hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/protocolPB/OMPBHelper.java

```diff
@@ -196,6 +216,165 @@ public static HddsProtos.DefaultReplicationConfig convert(
     return builder.build();
   }
 
+  public static FileChecksum convert(FileChecksumProto proto) {
+    if (proto == null) {
+      return null;
+    }
+    try {
+      switch (proto.getChecksumType()) {
+      case MD5CRC:
+        if (proto.hasMd5Crc()) {
+          return convertMD5MD5FileChecksum(proto.getMd5Crc());
+        } else {
+          LOG.warn("The field md5Crc is not set.");
```

Comment:
Shall we throw exception when there is no related checksum data included？

Reply:
make sense. will update